caravel/assets/javascripts/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -137,6 +141,10 @@ class TabbedSqlEditors extends React.PureComponent {
                 <i className="fa fa-clipboard" /> <CopyQueryTabUrl queryEditor={qe} />
               </MenuItem>
             }
+            <MenuItem eventKey="4" onClick={this.toggleLeftBar.bind(this)}>
+              <i className="fa fa-cogs" />
```

Comment:
we should add an `&nbsp`  here so there is a space between the icon and the text.